src/main/java/Tuesday/command/DeleteCommand.java

```diff
@@ -0,0 +1,83 @@
+package Tuesday.command;
+
+import Tuesday.util.Storage;
+import Tuesday.task.Task;
+import Tuesday.util.Ui;
+
+import java.io.*;
```

Comment:
Perhaps you may want to list out only the specific packages you would be using?

Reply:
I did not realise this. Thank you